tmva/sofie/inc/TMVA/ROperator_Concat.hxx

```diff
@@ -98,7 +98,7 @@
          }
 
          // get shape of output given inputs. It is going to be called after initialized
-         std::vector<std::vector<Dim>> ShapeInference(const std::vector<std::vector<Dim>> & inputs){
+         std::vector<std::vector<Dim>> ShapeInference(const std::vector<std::vector<Dim>> & inputs) {
```

Comment:
Do we need override here?

Reply:
No, it is not needed, because it is not defined in the ROperator class with this signature